app/core/vueRouter.js

```diff
@@ -29,6 +31,15 @@ export default function getVueRouter () {
           component: PageCinematicEditor,
           props: true
         },
+        {
+          path: '/editor/cutscene',
+          component: PageCutsceneEditorList
+        },
+        {
+          path: '/editor/cutscene/:slugOrId',
```

Comment:
Nitpick: We can use a single path for `/editor/cutscene` and then subpath for `/:slugOrId` similar to school admin routes.

Reply:
That's true. Does this require an additional refactor to use `<router-view>`?
I might punt this refactor.